jobs/build/ocp/Jenkinsfile

```diff
@@ -25,6 +25,7 @@ online:stg                {origin,origin-web-console,openshift-ansible}/stage ->
                           [$class: 'hudson.model.BooleanParameterDefinition', defaultValue: false, description: 'Mock run to pickup new Jenkins parameters?', name: 'MOCK'],
                           [$class: 'hudson.model.BooleanParameterDefinition', defaultValue: false, description: 'Run as much code as possible without pushing / building?', name: 'TEST'],
                           [$class: 'hudson.model.TextParameterDefinition', defaultValue: "", description: 'Include special notes in the build email?', name: 'SPECIAL_NOTES'],
+                          [$class: 'hudson.model.TextParameterDefinition', defaultValue: "", description: 'Exclude these images from builds. Comma or space separated list', name: 'BUILD_EXCLUSIONS'],
```

Comment:
If you don't permit newlines, use StringParameterDefinition to ensure people don't get tempted.